modules/node_modules/@ncigdc/containers/FrequentlyMutatedGenesTable.js

```diff
@@ -145,6 +146,11 @@ const FrequentlyMutatedGenesTableComponent = compose(
             ),
           },
           {
+            key: 'annotations',
+            title: 'Annotations',
+            style: { padding: '5px 0 0 0', textAlign: 'center' },
```

Comment:
thoughts on this https://github.com/NCI-GDC/portal-ui/pull/1542/commits/d13565bd78ea4feeb0f7f8e679d058604b10d445 as a stop gap fix before table redesign. the `padding: '5px 0 0 0'` before was because `text-align: right` on # Mutations th made that title too close to `Annotations`